### DISCUSSION TITLE
Rename `Actor::Get*` to `Actor::TryGet*` to match C# naming conventions

### DISCUSSION THREAD

**Discussion by riddlemd** [nxrighthere/UnrealCLR#81] [Ideas]
I know it would be an api breaking change for people currently using the plugin, but would be more inline with c# convention.

Also perhaps use out vs ref, so inline declarations of the variable could be made?

**Comment by nxrighthere** [comment]
Breaking changes are acceptable as long as the plugin in the preview stage. I'll look into that.
